code/ui/InfoFeed.cs

```diff
@@ -0,0 +1,29 @@
+namespace Sandbox.UI
+{
+    public partial class InfoFeed : Panel
```

Comment:
Got a better name for this?

Reply:
InfoFeed would match the upper right panel in name which was called MessageStack in TTT2. Idk ^^